index.html

```diff
@@ -54,7 +54,7 @@ <h1 class="primary-heading">Nurgul Kereikhan</h1>
             <img
               class="project-img"
               src="./assets/images/Rectangle21.svg"
-              alt="Project-1"
+              alt="There is laptop image"
             />
```

Comment:
Kindly keep the alt text brief as described above 🤗

A better alt text could be `Laptop` since the screen readers will read it as `image of Laptop` as described [here](https://www.w3.org/WAI/test-evaluate/preliminary/#images)
![image](https://user-images.githubusercontent.com/73714615/188656042-4a32f9db-ded3-438e-8da9-5a9a2017afa9.png)